src/tools/rbd/action/Trash.cc

```diff
@@ -178,14 +179,20 @@ int execute_remove(const po::variables_map &vm,
   return r;
 }
 
-std::string delete_status(time_t deferment_end_time) {
+std::string delete_status(time_t deferment_end_time, std::string state) {
   time_t now = time(nullptr);
 
   std::string time_str = ctime(&deferment_end_time);
   time_str = time_str.substr(0, time_str.length() - 1);
 
   std::stringstream ss;
-  if (now < deferment_end_time) {
+  if (state == "RESTORING") {
```

Comment:
Nit: use enum switch here since converting to an enum to a string below and then testing the string for a specific value here before outputing the same string again is a bit odd.